modules/about/Socials.module.css

```diff
@@ -1,8 +1,14 @@
 .socialsContainer {
   display: grid;
-  grid-template-columns: repeat(5, 1fr);
+  grid-template-columns: repeat(3, 1fr);
   padding-top: 4rem;
   width: fit-content;
   gap: 24px;
   margin: auto;
 }
+
+@media screen and (min-width: 600px) {
+  .socialsContainer {
+    grid-template-columns: repeat(5, 1fr);
```

Comment:
I think it's more consistent to keep all links in one row and reduce the gap between them:
```css
.socialsContainer {
  grid-template-columns: repeat(5, 1fr);
  gap: 0;
}

@media screen and (min-width: 600px) {
  .socialsContainer {
    gap: 24px;
  }
}
```
It fits perfectly in 360px wide screens, and if we want to support smaller screens we could reduce a tiny bit the padding around the links.

| before | after |
| --- | --- |
| <img width="368" alt="Capture d’écran 2023-03-01 à 19 32 27" src="https://user-images.githubusercontent.com/22420399/222232476-d4923030-c30a-4070-9353-1b8954b44464.png"> | <img width="369" alt="Capture d’écran 2023-03-01 à 19 32 00" src="https://user-images.githubusercontent.com/22420399/222232232-433af830-c680-4e2d-9505-11657811ec99.png"> |